docs/data-management.rst

```diff
@@ -226,6 +232,87 @@ Typically an administrator works on behalf of other users in a group where the a
+
+In case of File Attachment objects though, which also have files to them, each duplication will duplicate the linked file, thus doubling the storage space necessary for these File Attachment files.
+
+Resources
+---------
+
+-  Necessary software versions:
+
+   - **OMERO.server 5.6.3 or later**
+   - **omero-cli-duplicate plugin 0.4.0 or later**
+
+-  Documentation:
+
+   - `README of the omero-cli-duplicate repository <https://github.com/ome/omero-cli-duplicate/blob/master/README.rst>`_
+
+   - `Application Programming Interface documentation <https://docs.openmicroscopy.org/omero-blitz/5.5.7/slice2html/omero/cmd/Duplicate.html>`_
+
+-  Plugin for duplication on Command Line:
+
+   - `omero-cli-duplicate on PyPI <https://pypi.org/project/omero-cli-duplicate/#description>`_
+
+
+Setup
+-----
+
+**Duplicate plugin installation**
+
+- Go to the environment where you installed your OMERO.cli as specified under -  https://docs.openmicroscopy.org/latest/omero/users/cli/installation.html.
+
+- Activate the virtual environment.
+
+- Run::
+    
+   $ pip install omero-cli-duplicate
+
+Step-by-Step
+------------
+
+#. On your local machine, open a terminal
+
+#. Activate the virtual environment where ``omero-py`` is installed or add it to ``PATH`` e.g.::
+    
+   $ export PATH=/opt/omero/server/venv3/bin:$PATH
+
+#. The variables ``$ID​1`` and ``$ID2`` below are the IDs of the ​selected Datasets. To duplicate two Datasets with their Images and annotations on both the Images and the Datasets, run::
+    
+   $ ​omero duplicate Dataset:$ID1,$ID2 --report
+
+#. The duplicated Datasets will not be linked to any Project, even if the originals were linked to some Project. 
+
+#. Duplicate two Images with many ROIs on them. The ROIs duplication might take a long time. To exclude the duplication of the ROIs, run::
+
+   $ omero duplicate Image:$ID1,$ID2 --ignore-classes=Roi --report 
+
+#. Find the duplicated Images in the Orphaned Images and Drag and Drop them into a Dataset or create a new Dataset for them.
+
+#. Duplicate two Projects of another user in read-annotate group type. The group name in our example below is `read-annotate-group`. Log in as `user-1` and duplicate Projects belonging to `user-2` in that group. Further, instead of duplicating all the annotations of `user-2`, link all the original annotations such as Tags, Key-Value pairs and FileAnnotations of `user-2` to the corresponding objects in the duplicate, and duplicate only Comments and Ratings. The duplicate and Comments and Ratings on it will belong to `user-1`, but the other annotations will still belong to `user-2`. Run::
```

Comment:
Suggested re-write (hopefully not too verbose)?

Duplicate two Projects of another user in read-annotate group type. Here we specify some `reference-classes` of objects that we do not want to duplicate, even though they are linked to the objects we are duplicating. Instead, the duplicated objects will be linked to these 'reference-classes' of object.
We can also specify classes that we explicitly do want to duplicate using the `duplicate-classes` argument.
In this example, we specify all Annotations as `reference-classes`, but a subset of these (Comments and LongAnnotations such as ratings) are `duplicate-classes`. This means that the duplicated Projects will be linked to the original annotations such as Tags, Key-Value pairs and FileAnnotations, but Comments and Ratings will be duplicated. This can be useful to ensure that a single Comment is not attached to multiple objects which might cause confusion when the comment is edited. The duplicated annotations will belong to the current user (user-1), even if all the original annotations may belong to other users in the read-annotate group. Run::

Reply:
> even if all the original annotations may belong to other users in the read-annotate group.

Yes, thanks, this is more precise. I did not think through the mixed annotation ownership on the original.